jobs/v3/api_client/auto_complete_sample.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
Per legal's advice, headers of all post-2016 samples use `2018 Google LLC` instead of `2018 Google Inc.`.